### DISCUSSION TITLE
Add post tags

### DISCUSSION THREAD

**Discussion by simonv3** [funmusicplace/mirlo#606] [Ideas]
Add a table called `PostTags` that allows users to add tags to posts specifically. TBD if these should be linked to music tags or something different

**Comment by gribbles67** [comment]
It makes sense to be able to tag posts with the same tags as music so you can also search/group/filter on genre and see both music of that genre and posts related to that genre.

Plus I guess you then get the option to search for posts related to  #trance #review or #punk #release, etc etc etc

Add a toggle in the search to allow the user to search for music or posts (or whatever other media may eventually pop up - merch?)

**Comment by Cwpute** [comment]
Could be interesting to see what artists are posting about.
From a quick look, it seems be mostly Upcoming releases, new releases, upcoming concerts, past concerts, general news about what's going on, and on Mirlo i've seen thought-pieces as well, as some artists seem to fully embrace the "blog" aspect of it all !
